tests/.pylintrc

```diff
@@ -3,7 +3,7 @@ extension-pkg-whitelist=PyQt5
 
 [BASIC]
 argument-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
-attr-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
+attr-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*)|(?:[A-Z][a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
```

Comment:
😂 Oh..
What would you prefer: `self.MockedThread` vs. `self.mocked_thread_cls`?

Reply:
I inteded to meet the PyQt class name convention, but I don't like it more that I changed the regex rule..
Thus, I will adopt the latter!